sdk/core/azure-core-tracing-opentelemetry/test/ut/CMakeLists.txt

```diff
@@ -15,7 +15,7 @@ endif()
 
 if (CMAKE_CXX_COMPILER_ID MATCHES "MSVC")
 # Disable deprecation warnings.
-  add_compile_options(/wd4996)
+#  add_compile_options(/wd4996)
```

Comment:
unintended change?

Reply:
Good catch - intended change.

Disabling the 4996 warning in a cmakelists.txt file is almost always a coding mistake (because it hides warnings), so I'm trying to clean up all instances of them.